yt_dlp/extractor/zaiko.py

```diff
@@ -90,3 +92,27 @@ def _real_extract(self, url):
                 'thumbnail': ('poster_url', {url_or_none}),
             })),
         }
+
+
+class ZaikoETicketIE(ZaikoBaseIE):
+    _VALID_URL = r'https?://zaiko\.io/account/eticket/(?P<id>\w{48}=)'
+    _TESTS = [{
+        'url': 'https://zaiko.io/account/eticket/000000000000000000000000000000000000000000000000=',
+        'only_matching': True,
+    }]
+
+    def _real_extract(self, url):
+        ticket_id = self._match_id(url)
+
+        webpage, urlh = self._download_webpage_handle(url, ticket_id)
+        final_url = urlh.geturl()
+        if 'zaiko.io/login' in final_url:
+            self.raise_login_required()
```

Comment:
take the similar code from `ZaikoIE` and move it to a baseclass method, `_download_real_webpage` or something